<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountPageClass</class>
 <widget class="QMainWindow" name="AccountPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AccountPage</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Poze/Resources/1.jpg);</string>
   </property>
   <widget class="QLabel" name="usernameLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: transparent;</string>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="matchesPlayed">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: transparent;</string>
    </property>
    <property name="text">
     <string>Matches played:</string>
    </property>
   </widget>
   <widget class="QLabel" name="usernameEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: transparent;</string>
    </property>
    <property name="text">
     <string>inputUsername</string>
    </property>
   </widget>
   <widget class="QLabel" name="matchesPlayedEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: transparent;</string>
    </property>
    <property name="text">
     <string>inputMatchesPlayed</string>
    </property>
   </widget>
   <widget class="QLabel" name="matchesWon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: transparent;</string>
    </property>
    <property name="text">
     <string>Matches won:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Level">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: transparent;</string>
    </property>
    <property name="text">
     <string>Level:</string>
    </property>
   </widget>
   <widget class="QLabel" name="matchesWonEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: transparent;</string>
    </property>
    <property name="text">
     <string>inputMatchesWon</string>
    </property>
   </widget>
   <widget class="QLabel" name="levelEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: transparent;</string>
    </property>
    <property name="text">
     <string>inputLevel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Registration.qrc"/>
 </resources>
 <connections/>
</ui>
